packages/styles/scss/patterns/listsection/README.md

```diff
@@ -0,0 +1,40 @@
+### SCSS
+
+#### Usage
+
+Import the package css into the top of your main CSS file.
+
+```css
+@import '@carbon/ibmdotcom-styles/scss/patterns/listsection/index';
+```
+
+#### Elements
+
+| Class                                     | Description                                                                            |
+| ----------------------------------------- | -------------------------------------------------------------------------------------- |
+| `.bx--listsection__container`             | Element containing all list section content                                            |
+| `.bx--listsection__row`                   | Vertical divisions across the viewport                                                 |
+| `.bx--listsection__col`                   | Horizonal divisions across the viewport and change in width at different screen widths |
+| `.bx--listsection__title`                 | Title for list section pattern                                                         |
+| `.bx--listsection__content`               | Content holder for short description                                                   |
+| `.bx--listsection__listsection-group_col` | Horizonal division for listgroup element                                               |
+| `.bx--listsection__divider_col`           | Horizonal division for divider element                                                 |
+| `.bx--listsection__divider`               | Divider element                                                                        |
+| `.bx--listsection-group__col`             | list section group column element                                                      |
```

Comment:
looks good, just have to clean up some of the documentation for classes that were removed

Reply:
right